lib/llvmopencl/Flatten.cc

```diff
@@ -75,12 +76,19 @@ Flatten::runOnModule(Module &M)
       if (KernelName == f->getName() || 
           (KernelName == "" && pocl::Workgroup::isKernelToProcess(*f)))
         {
-          AttributeSet attrs;
-          f->removeAttributes(
-              AttributeSet::FunctionIndex, 
-              attrs.addAttribute
-              (M.getContext(), 
-               AttributeSet::FunctionIndex, Attribute::AlwaysInline));
+#if LLVM_OLDER_THAN_5_0
+        AttributeSet attrs;
```

Comment:
Is there an indentation problem here? Also pls rename 'attrs' to Attrs as per LLVM coding style.